src/Raven.Client/Documents/Operations/TransactionsRecording/ReplayTransactionsRecordingOperation.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Converters;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.TransactionsRecording
+{
+    public class ReplayTransactionsRecordingOperation : IMaintenanceOperation<ReplayTrxOperationResult>
```

Comment:
As written below. This needs to return the OperationIdResult

Reply:
https://github.com/ravendb/ravendb/pull/7397#discussion_r212181012